config/webpack.config.js

```diff
@@ -276,12 +276,15 @@ module.exports = async (env = {}) => {
     buildOptions.destination,
   );
 
+  const publicPathHost = BUCKETS[buildtype] || '';
+  const publicPath = `${publicPathHost}/generated/`;
```

Comment:
we've tried this before and it ends up pointing CI to the production deployed assets instead of the local ones because it generates a full path in CI

Reply:
@timwright12 yeah, I had a feeling it was just reproducing what you had tried before but wasn't sure how to test that locally

Reply:
Review instances are actually a localhost build, so those are fine. The problem is with the production build in CI. When we set the publicPath to a full S3 bucket you get loading from `https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/` so when the E2E test run they run with the asset sitting in the S3 bucket rather than the local asset that may contain changes.

Our solution to this once before was to run the build twice. Once with the local assets (for the tests), then again with the s3 path, but it was adding too much time to the build.

I honestly wouldn't spend too much more time on this, it's a nice to have, but not a hard requirement